src/EmployerAan/SFA.DAS.EmployerAan.Api/Controllers/CalendarsController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Globalization;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerAan.Application.Calendars.Queries.GetCalendars;
+
+namespace SFA.DAS.EmployerAan.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class CalendarsController : Controller
```

Comment:
This should be derived from ControllerBase

Reply:
Done